### ISSUE TITLE
 [Android Bootcamp] Project 1 - Review my app #2 

### ISSUE THREAD

**Issue by MiguelBennu** [MiguelBennu/SimpleTodo#1]
My app is complete, please review. /cc @codepathreview @codepath

**Comment by nesquena** [comment]
Hi Luis Miguel,

Thanks for applying and submitting your pre-work. Based on your LinkedIn it is unclear if you meet the selection criteria for participating in the course. In particular:
- 1+ years of professional software development experience in a single role (shipping code to production and collaborating on a team strongly preferred)
- Bachelors or Masters Degree in CS/Engineering (OR additional 2+ years of professional experience)

Can you review that and let me know if you can confirm that you believe you meet the requirements? If not, I can still add you as a "remote self-paced observer" so you will still have full access to the curriculum and can learn at your pace. In the meantime, you should also check out the [resources we provide here as well](http://courses.codepath.com/snippets/intro_to_android/email_faq#heading-i-dont-have-much-or-any-professional-software-development-experience-how-can-codepath-help-me).

**Comment by MiguelBennu** [comment]
Nathan,

Thanks for your note, If my memory doesn't fail me I believe you were the
one who inspired me into applying to code path at one of the Android meet
ups.

Answers to your two questions:

1) Yes I have 1+ years of experience of professional software development
of shipping code.. My software development work experience has been built
mostly working on audio related products (things that have speakers and
microphones), so I apologize for the confusion.  In the audio product space
team collaboration is imperative due integration of complementary
disciplines, Digital signal processing, user interface, etc. and throughout
the entire software development life cycle, before and after the code is
released.

Are there any limitations as far a the type of industry in which the
software development experience has been obtained?

2) I thought my education was disclosed on Linked but in case it is not
obvious:

EDUCATIONAL QUALIFICATIONS
Master of Science, University of Miami,
Computer Engineering – Intelligible Systems and Communications

Bachelor, University of Miami,
Audio (Electrical) - Engineering

AA, Miami Dade College,
Pre-Engineering Course work, Sound Engineering and Music Coral Gables, Fl.

BTW, I am not opposed to a remote self-paced observer status as it would be
an honor to be involved even if in the sidelines but would like to be
considered and given an opportunity if possible, I think that the boot-camp
is ideal for me and I intend to give it my best. Please let me know if I
should continue to work on the submission and thanks for the links (already
reviewing them).

BR,

LMA

2016-06-09 17:29 GMT-07:00 Nathan Esquenazi notifications@github.com:

**Comment by nesquena** [comment]
Thanks for the explanation Luis Miguel. 

> Thanks for your note, If my memory doesn't fail me I believe you were the
> one who inspired me into applying to code path at one of the Android meet
> ups.

Yes! I thought you looked / sounded familiar. Thanks for applying!

> BTW, I am not opposed to a remote self-paced observer status as it would be
> an honor to be involved even if in the sidelines but would like to be
> considered and given an opportunity if possible, I think that the boot-camp
> is ideal for me and I intend to give it my best. Please let me know if I
> should continue to work on the submission and thanks for the links (already
> reviewing them).

For this particular pilot, due to a high volume, we are hoping to target participants with a bit more web or other application experience but I like to keep an open mind in terms of participants. Each round we select 5 or so participants that do not map to that background which is why we keep the application open. In particular, we select participants largely based on the quality and completeness of the pre-work. As a result, I would encourage you to continue to work on your submission :)

That said, either way, I'll be sure to provide you access to the course as a remote self-paced observer once the course starts, so you'll have access regardless. Thanks for your understanding.

**Comment by MiguelBennu** [comment]
Nathan,

Hey, just in case I do have WEB and APP development experience (1 + years).
Let me know if you need me to expand or provide names of references and
phone numbers.  I understand clearly that you are looking for a specific
profile but then based on your email it sounds like you guys need to be
more specific with the recruiting statements. I applied with the
understanding that this is a boot camp for existing developers that would
like to enhance their android application skills (ideal for _existing
developers_ who want to transition into Android development).

Here is the list of requirements I received via email and also is posted on
the site:
- Can be physically present for every onsite session in San Francisco (1
  excused absence)
- Can dedicate 10-15+ hours each week to the program
- 1 or more year(s) of professional software development experience
- Bachelors or Masters Degree in CS/Engineering (OR additional 2+ years
  of professional experience)
- Existing knowledge of Java or ability to ramp up on the language
  quickly
- Interested in collaborating with a team on a group project

So I checked all those boxes and applied.

In addition to this my current work is APP development at a start up in
Downtown SF, which is the reason why I went to the meet up near the TL, I
was looking for resources to enhance my knowledge. I firmly believe that
this boot camp could help me steer the project in the right direction.

If you are in San Francisco and would like to stop by so that you can put a
face to the name and further verify I would be happy to host you. That way
I can show you what I am working on even though you would probably consider
it bit boring as most people do, our focus is mental health amongst female
teenagers through an incubator program by PFIZER. Our solution is a mobile
APP and I am the guy in charge of the Development.

In any case, I will continue to fulfill the requirements and keep my hopes
up that in all fairness I will be considered for this opportunity and a
chance position in the boot camp.

Happy Friday!

LMA

in charge of

2016-06-09 22:19 GMT-07:00 Nathan Esquenazi notifications@github.com:

**Comment by nesquena** [comment]
> I applied with the
> understanding that this is a boot camp for existing developers that would
> like to enhance their android application skills (ideal for _existing
> developers_ who want to transition into Android development).

Yes, that is correct and the eligibility requirements you copied over are correct for eligibility into the course. Given that and the corrections you've made, you are eligible for the course and I'm glad you applied and I understand your interest in being a participant. 

> That way
> I can show you what I am working on even though you would probably consider
> it bit boring as most people do, our focus is mental health amongst female
> teenagers through an incubator program by PFIZER. Our solution is a mobile
> APP and I am the guy in charge of the Development.

That sounds like a really interesting project. 

> In any case, I will continue to fulfill the requirements and keep my hopes
> up that in all fairness I will be considered for this opportunity and a
> chance position in the boot camp.

I appreciate the the clarifications here. We'll keep this all in mind and you are absolutely eligible for the course and as I mentioned a big part of our final admissions relies on the pre-work. 

Apologies for any frustration here on your part, I hope you can appreciate that we received about 700+ applications and counting this round and so we necessarily have to do some selection above baseline eligibility and evaluate applicant's prior professional experience (LinkedIn, application). We've run this course for over a thousand engineers and we've learned a tremendous amount running these for the last 4 years. Our primary goal at the end of the day is to select a diverse group of 30 in-person participants from hundreds of applicants that will get the most out of the course and have the highest chance of an awesome experience heavily collaborating together with mentors and each other over eight weeks.

**Comment by MiguelBennu** [comment]
Nathan,

No need to apologize, no frustration here. I firmly believe that
communication is the key to building a successful network and very often we
don't voice our opinions/questions due fear of the outcomes. I simply
wanted to make sure I was following the right approach, channels and and at
the same time providing more insight for further consideration.

I thank you for your email, hopefully will run across again,

LMA

2016-06-10 11:27 GMT-07:00 Nathan Esquenazi notifications@github.com:

**Comment by nesquena** [comment]
I completely agree. Thanks and we'll keep an eye out for any revisions to your pre-work, ping us back here when you want another review. Best.
